### DISCUSSION TITLE
Set Custom Line Numbers

### DISCUSSION THREAD

**Discussion by olaws** [VPKSoft/ScintillaNET-FindReplaceDialog#9] [Ideas]
ScintillaNET has the ability to set custom line numbers, [see here](https://github-wiki-see.page/m/jacobslusser/ScintillaNET/wiki/Displaying-Line-Numbers). However, I haven't been able to sync these custom line numbers with ScintillaNET-FindReplaceDialog's FindAllResultsPanel and GoToDialog. Can this be added please?

**Comment by VPKSoft** [comment]
Hi,
The functionality can probably be added to both by adding some kind of event requesting a "real" line index for the user input. That being said, I am not going to implement such (due to lack of personal interest in the enhancement) but I will always welcome a PR if someone wished to implement a custom line number support for these dialogs. 👍
